controllers/managedfusion_controller.go

```diff
@@ -681,8 +699,8 @@ func (r *ManagedFusionReconciler) reconcileMonitoringResources() error {
 	}
 
 	serviceMonitorList := promv1.ServiceMonitorList{}
-	if err := r.list(&serviceMonitorList); err != nil {
-		return fmt.Errorf("Could not list service monitors: %v", err)
+	if err := r.list(&serviceMonitorList, client.InNamespace(r.Namespace)); err != nil {
```

Comment:
Same as above

Reply:
I have reverted the changes to be moved to `datafoundation.go` at a future Pr